packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -98,6 +100,34 @@ const Text = styled.div`
     position: relative;
 `;
 
+const TextHover = styled.div`
+    position: relative;
+    &:after {
+        content: '';
+        position: absolute;
+        top: -4px;
+        left: -4px;
+        right: -4px;
+        bottom: -4px;
+        border-radius: ${variables.BORDER_RADIUS.HOVER};
+        transition: all ${variables.HOVER_TRANSITION.DURATION}
+            ${variables.HOVER_TRANSITION.ANIMATION};
+        background-color: transparent;
+    }
+
+    &:hover,
+    &:focus,
+    &:active {
+        &:after {
+            background-color: ${props => props.theme.BG_HOVER_ITEM};
+            top: -8px;
+            left: -8px;
+            bottom: -8px;
+            right: -8px;
+        }
+    }
+`;
+
```

Comment:
I'll check buttons' padding and fix it.

What do you mean is over-engineering - using pseudo element or some other stuff? I used that as it seems to be the simplest solution to how to avoid button padding/margin changing. Maybe we can create a wrapping component with these styles and apply it everywhere we want it?

I'm not sure about background color in not hovered state as I was inspired by [this prototype](https://www.figma.com/proto/BI5NMueIpbxfc3CMYUQaEW/Exploring-Hovers?page-id=2%3A3&node-id=201%3A102&viewport=-1566%2C861%2C0.7228973507881165&scaling=min-zoom). Probably should be discussed with designer?

I'm also open for discussion how to unify buttons with links and its styles

Reply:
Let's open discussion about that on standup